### PR TITLE
Cart and wishlist page added

### PR DESCRIPTION
**Pull request by Anurag021**
Added Cart and whishlist page in ecommerce

### added files
- cart.css
- wishlist.css
- cart.html
- wishlist.html
### Preview Link
cart page --> https://hardcore-sinoussi-9e4133.netlify.app/cart-and-wishlist/cart
wishlist page --> https://hardcore-sinoussi-9e4133.netlify.app/cart-and-wishlist/wishlist

### CODE REVIEW

**Comment by shahbazibnafzal** [comment]
Hi Anurag, good job.
just one suggestion your logo and search bar could be in a row. 

image

File: Components/wishList.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.wish-list-container{
+    margin-left: 6rem;
+    flex-wrap: wrap;
+    width: 70rem;
+    height: auto;
+    margin-bottom: 1rem;
+}
```
Comment (ApurvaSawant11) [inline Components/wishList.css:7]: Wishlist is on the left of the page. Align this container to center.

File: Components/wishList.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+.whishlist-btn{
+    margin: 2px;
+    border-radius: 10px;
+}
+.vertical-card-button > .whishlist-btn{
+    padding: 3px;
+}
+.vertical-card-button:hover > .whishlist-btn{
+    padding: 3px;
+}
```
Comment (ApurvaSawant11) [inline Components/wishList.css:24]: Properties are same for both of them. Use comma separator like 
`.vertical-card-button > .whishlist-btn, 
.vertical-card-button:hover > .whishlist-btn{
    padding: 3px;
}`

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

**Review by shahbazibnafzal — APPROVED** [review]
Looks good

**Review by arjun-computer-geek — APPROVED** [review]

File: Components/wishList.css (hunk 2 above)
Comment (Anurag021) [reply Components/wishList.css:24]: Thanksfor poinint it out,  implemented.

**Pull request merged**